gradle.properties

```diff
@@ -8,3 +8,6 @@ archivesBaseName=id-images-api
 groovyVersion=2.4.3
 dropwizardVersion=0.8.2
 codenarcVersion=0.21
+
+# contrib repository properties
+contribRepo=git@github.sig.oregonstate.edu:ecs-data/id-images-api-contrib.git
```

Comment:
Can we use some name conventions in the contrib repo name? For example instead of specifying again "id-images-api" at the end, can we reference the "archivesBaseName" from earlier in this file or some other variable?